__tests__/app.test.js

```diff
@@ -25,3 +25,26 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const body = res.body;
+        expect(Array.isArray(body)).toBe(true)
+        expect(body.length).toBe(3);
+        expect(typeof body[0].slug).toBe("string")
```

Comment:
We wouldn't just expect our topics to have a slug, but also a description - please add an assertion for that, too!